server_utils/origin.go

```diff
@@ -38,18 +38,18 @@ var originExports []OriginExport
 
 type (
 	OriginExport struct {
-		StoragePrefix    string `json:"storage_prefix"`
-		FederationPrefix string `json:"federation_prefix"`
+		StoragePrefix    string `json:"storagePrefix"`
```

Comment:
I'm guessing it's the registry that normally ingests these JSON values? Does changing the cases break any backwards compatibility with an old registry?

Reply:
No, the JSON of this struct is only used for UI. We have different structs for registry/director to ingest